lib/decompress/zstd_decompress.c

```diff
@@ -225,6 +225,8 @@ size_t ZSTD_sizeof_DCtx (const ZSTD_DCtx* dctx)
            + dctx->inBuffSize + dctx->outBuffSize;
 }
 
+size_t ZSTD_sizeof_DCtxWindow(const ZSTD_DCtx* dctx) { return dctx->fParams.windowSize; }
```

Comment:
I don't think there is a need for a new library symbol to deliver this information.
Look at `ZSTD_getFrameHeader()`.